plugin/trino-kafka/pom.xml

```diff
@@ -339,6 +339,46 @@
                     </ignoredResourcePatterns>
                 </configuration>
             </plugin>
+            <!-- TODO: Instead of using this plugin to generate classes we should invoke the parser directly (https://github.com/trinodb/trino/issues/16039) -->
+            <plugin>
+                <groupId>com.github.os72</groupId>
+                <artifactId>protoc-jar-maven-plugin</artifactId>
```

Comment:
Let's use a specific  version of this plugin - https://github.com/trinodb/trino/pull/16043

Reply:
Thank you.